src/main/java/seedu/address/logic/parser/InteractionCommandParser.java

```diff
@@ -46,19 +45,19 @@ public Command parse(String args) throws ParseException {
         if (argMultimap.getValue(PREFIX_OUTCOME).isPresent()) {
             String outcomeString = argMultimap.getValue(PREFIX_OUTCOME).get().split("\\s+")[0];
             outcome = ParserUtil.parseOutcome(outcomeString);
-            String[] trimmedArgsParts = trimmedArgs.split("\\s+", 4);
-            if (trimmedArgsParts.length < 4) {
+            String[] trimmedArgsParts = trimmedArgs.split("\\s+", 3);
+            if (trimmedArgsParts.length < 3) {
                 note = "";
```

Comment:
nit: we can invert this `if` statement right, since `note` is initialized to an empty string

Reply:
Done